plugins/lookup/conjur_variable.py

```diff
@@ -278,6 +278,9 @@ def run(self, terms, variables=None, **kwargs):
         conf_file = self.get_option('config_file')
         as_file = self.get_option('as_file')
 
+        if validate_certs is False:
+            display.warning('Certificate validation has been disabled. Please enable with validate_certs option.')
```

Comment:
Do you know if we validate that the `appliance_url` uses HTTPS? Maybe that would be a good addition at this stage.